ExampleMod/Content/Tiles/ExampleSink.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ObjectData;
+
+namespace ExampleMod.Content.Tiles
+{
+	public class ExampleSink : ModTile
+	{
+		public override void SetDefaults()
+		{
```

Comment:
This and the other example have the wrong brace style. Make your editor use the provided editorconfig.

Reply:
fixed